cpp/ps0001/ps0001_sol.cpp

```diff
@@ -4,15 +4,19 @@ using namespace std;
 
 int main()
 {
-    int a ,b ,c ;
-    int s ;
-    cout<< "Enter first number \n" ;
-    cin>> a;
-    cout<< "Enter second number \n" ;
-    cin>> b ;
-     cout<< "Enter third number \n" ;
-     cin>> c ;
-     s= a + b + c ;
-     cout<< " The sum is "<< s << endl ;
+    // prompting users for 3 numbers to add 1/2
+    unsigned short a, b, c;
+    unsigned short s;
+    cout << "Enter first number \n";
+    cin >> a;
+    cout << "Enter second number \n";
+    cin >> b;
+    cout << "Enter third number \n";
+    cin >> c;
+    
+    // outputting the result of the summation process 3/3
+    s = a + b + c ;
+    cout << "The sum is " << s << endl;
     return 0;
 }
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [ ] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code